sdk/rust/oak/src/io/mod.rs

```diff
@@ -103,7 +103,9 @@ pub fn entrypoint_node_create<
 ) -> Result<Sender<Command>, oak_io::OakError> {
     let node_config = &crate::node_config::wasm(wasm_module_name, T::ENTRYPOINT_NAME);
     let init_sender = node_create(name, label, node_config)?;
-    send_init(init_sender, init, label)
+    let result = send_init(&init_sender, init, label);
+    init_sender.close()?;
```

Comment:
I understand that this is meant to close the channel even if the sending failed, but I wonder in practice when that may happen. Would it be the case if the labels are not correct, for instance? Is this fine, or should we think of some other way of doing it to make sure channels get closed? Apart from the linear handles proposal, which I guess would solve it anyways.

Reply:
I tried to implement `Drop` for some structs, but I could not make it to work. I cannot think of any other general solution.  Perhaps we should revisit these changes after the work on linear handles is completed.